Fruit_Jam/Larsio_Paint_Music/sound_manager.py

```diff
@@ -98,9 +106,24 @@ def __init__(self, audio_output="pwm", seconds_per_eighth=0.25):
 
                 # Initialize TLV320
                 self.tlv = adafruit_tlv320.TLV320DAC3100(i2c)
+                self.tlv.reset()
```

Comment:
Does the TLV320 need to get `reset()` here?  It should be trying to reset during the init function here: https://github.com/adafruit/Adafruit_CircuitPython_TLV320/blob/f24788d94ad5288fd26ba9b01ded8107ba943c86/adafruit_tlv320.py#L866-L867 Were you seeing issues with it that led to adding the reset() here?

Reply:
I added that when I was having problems controlling the volume before learning about dac_volume. A reset in the init should be sufficient, I'll get rid of this one.